src/calendar/index.js

```diff
@@ -272,6 +272,7 @@ class Calendar extends Component {
           weekNumbers={this.props.showWeekNumbers}
           onPressArrowLeft={this.props.onPressArrowLeft}
           onPressArrowRight={this.props.onPressArrowRight}
+          {...this.props}
```

Comment:
This is dangerous as you let the user override all props sent to CalendarHeader.